joomla/joomla-cms
# [plg_fields] Manifest cleanup, filters, Editor plugin fix

Pull Request for Issue # .

### Summary of Changes

Added filter attribute to some XML fields.
Fixed 'Show Buttons' option in Editor field.
Some CS (hidden tabs and spaces).

### Testing Instructions

1) Code review.

2) In `Fields - Editor` plugin set `Show Buttons` option to `Yes`. Create editor type field, set `Show Buttons` to `Use settings from plugin`. Edit an article, in `Fields` tab check the editor field.

### Expected result

2) Buttons are shown.

### Actual result

2) Buttons not shown.

### Documentation Changes Required
No.
